Print warning instead of throw if fail to find role to set up x-ray (#186)
DataDog/datadog-cloudformation-macro: serverless/README.md

## Configuration

> To further configure your macro, use the following custom parameters:

Author: Mostly auto formatting

This error can occur if the IAM user executing the command lacks the `lambda:InvokeFunction` permission. Add the permission to the user's IAM role.

### Setting up X-Ray tracing using a predefined Lambda execution role

> To enable X-Ray tracing when you use a predefined Lambda execution role (i.e. created outside the CloudFormation template), e.g.

Reviewer: ```suggestion
Complete the following steps to enable X-Ray tracing when using a predefined Lambda execution role (such as one created outside the CloudFormation template, like this example):
```

```
Resources:
  MyFunction:
    Type: AWS::Serverless::Function
    Properties:
      Role: !Sub arn:aws:iam::${AWS::AccountId}:role/PredefinedRole
```

> 
> please:

Reviewer: ```suggestion
```

> 1. Set `enableXrayTracing` option to `true` for the macro

Reviewer: ```suggestion
1. Set the `enableXrayTracing` option to `true` for the macro.
```
